composer.json

```diff
@@ -1,4 +1,20 @@
 {
+    "name": "lianahq/liana-client",
```

Comment:
Packagist is the defacto standard in PHP package management for many years. It would be much easier for developer to do composer installation than to download the zip and maintain within their own code base.

Even if not listed in packagist, the package name helps composer to install it directly from github with proper [repositories](https://getcomposer.org/doc/05-repositories.md#composer) setting.

Reply:
We prefer to keep this repository as a simple source from which people can copy-paste code if necessary. And since we are not planning to use Packagist at the moment I think it would be best not to have any related configurations pointing towards non-existing Packagist accounts as it now would seem to do.

Reply:
Composer supports installation of package directly from github (without any Packagist setup). If you only setup the "name" here, everything will work. As stated before, you do not need Packagist account for this setup.

Reply:
Copying the whole source code into one's own source code would risk missing security updates / feature patch from upstream vendor (i.e. you). Using composer is a key recommended practice in modern PHP development. As stated before, you can support modern development practice without Packagist. But you need to have a proper composer.json with "name" setup.